tutorials/quickstarts/environment.yml

```diff
@@ -0,0 +1,29 @@
+name: azureml_py36
+dependencies:
+  # The python interpreter version.
+  # Currently Azure ML only supports 3.5.2 and later.
```

Comment:
is all the pinning needed? that usually indicates a pretty fragile example

Reply:
Not sure, I got the file from https://github.com/Azure/MachineLearningNotebooks/tree/master/how-to-use-azureml/automated-machine-learning , as I though that must be the place to go for the correct version. I can check with the people that made it.